crates/cli-support/src/js/mod.rs

```diff
@@ -1768,6 +1768,7 @@ impl<'a> Context<'a> {
         self.memview("Float64", memory)
     }
 
+    #[allow(dead_code)]
```

Comment:
I had to add some `#[allow(dead_code)]` annotations to compile without warnings. `memview_function` does seem genuinely unused.

Reply:
Would you mind going ahead and just deleting it?